Research/Research/Core/Presentation/RSDTaskViewModel.swift

```diff
@@ -588,8 +619,27 @@ open class RSDTaskViewModel : RSDTaskState, RSDTaskPathComponent {
     }
 }
 
+extension RSDTaskResult {
+    public var didAbandon: Bool {
+        self.asyncResults?.contains(where: {
+            $0.identifier == RSDIdentifier.abandonAssessment.rawValue &&
+                ($0 as? AnswerResult)?.jsonValue == .boolean(true)
+        }) ?? false
+    }
+}
```

Comment:
@kalperin 
This extension method puts the `didAbandon` flag on the *result* using a look-up of the async results.